apps/condo/domains/common/components/SelectStyled/useApplySelectStyledCss.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/core'
+import { colors } from '../../constants/style'
+
+
+export const useApplySelectStyledCss = (applyCss: boolean, raisedClearButton: boolean) => {
+
+    const greyStyle = css`
```

Comment:
it's constant actually and can be moved outside the function

Reply:
OK, thx. Corrected